### PR TITLE
Updates python3 test to check for either python3 or python36 pkg names

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

File: tests/test_ami.py [hunk 1]
```diff
@@ -100,22 +100,49 @@ def test_el7_fips_disabled(host):
     assert not fips.exists or fips.content.strip() == b'0'
 
 
-@pytest.mark.parametrize("name", [
-    ("python36")
+@pytest.mark.parametrize("names", [
```
Comment (ferricoxide) [inline tests/test_ami.py]: So, pytest effectively treats this as an ORed test?

Comment (lorengordon) [reply tests/test_ami.py]: The logic in the function does, yes. We are just telling pytest to pass a specific list of packages to the function. The function loops over the items in the list, breaks if the package is installed, and then asserts the installation status. If none of the pkgs are installed, the assert fails and pytest fails the test.

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
@plus3it/spel this build is now passing, please review...

File: tests/test_ami.py (hunk 1 above)
Comment (ferricoxide) [reply tests/test_ami.py]: woot

**Pull request merged**